.github/workflows/build-push-latest.yaml

```
@@ -0,0 +1,106 @@
+
+      - name: run helm dependency update
+        working-directory: target/helm/dave-backend
+        run: helm dep update
+
+      - name: run helm package
+        working-directory: target/helm
+        run: helm package dave-backend
+
+      - name: get version number
+        working-directory: target/helm/dave-backend
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:81: Double quote to prevent globbing and word splitting

(shellcheck)